### ISSUE TITLE
aura -Br doesn't work

### ISSUE THREAD

**Issue by atondwal** [aurapm/aura#345]
```
~ ❯❯❯ sudo aura -Br                                                                           ⏎
aura >>= Saved package state.
```

```
~ ❯❯❯ aura --version

 __ _ _  _ _ _ __ _    Pacman v4.2.1 - libalpm v9.0.1
/ _` | || | '_/ _` |   Copyright (C) 2006-2014 Pacman Development Team
\__,_|\_,_|_| \__,_|   Copyright (C) 2002-2006 Judd Vinet
AURA Version 1.3.2.0   
 by Colin Woodbury     This program may be freely redistributed under
                       the terms of the GNU General Public License.
Aura Translators:      
 Chris "Kwpolska" Warrick (Polish)
 Denis Kasak / "stranac" (Croatian)
 Fredrik Haikarainen (Swedish)
 Lukas Niederbremer / Jonas Platte (German)
 Alejandro Gómez (Spanish)
 Henry "Ingvij" Kupty / Thiago "thiagowfx" Perrotta (Portuguese)
 Ma Jiehong / Fabien Dubosson (French)
 Kyrylo Silin (Russian)
 Bob Valantin (Italian)
 Filip Brcic (Serbian)
 "chinatsun" (Norwegian)
 "pak tua Greg" (Indonesia)
```

**Comment by flamusdiu** [comment]
@fosskers I would have to agree. All `-Br` is doing is creating another package state. Though, `-Bc` works fine. =)

**Comment by despairblue** [comment]
same thing here

**Comment by fosskers** [comment]
Odd, perhaps something got mixed up during some recent changes to how flags are parsed.

**Comment by atondwal** [comment]
Does anyone know when this _was_ working so that I can bisect it?

**Comment by flamusdiu** [comment]
@atondwal sadly, I don't ... =( I didn't even realize it till I saw this.

**Comment by despairblue** [comment]
Me neither, I just tried to show that feature to a friend, and well... :worried:

**Comment by stepvhen** [comment]
I am pretty sure this was working on 1.3.0.1, though I am not 100% on that.

**Comment by fosskers** [comment]
This is "fixed" with Aura 1.3.8. Use the long flag version: `aura -B --restore`.
